### ISSUE TITLE
Support for RX580

### ISSUE THREAD

**Issue by DmitriZamysloff** [nanopool/Claymore-XMR-Miner#2]
Hi, the miner is very good I am looking forward for the version, which supports RX580 under windows. Hope it can get more hashes, then actual miner, which recognises my GPUs as RX480.

**Comment by weyds** [comment]
+1

**Comment by sothcott** [comment]
I don't think it matters. It's pretty much the same card with some speed improvements. You should still be able to get the maximum from it.
